### ISSUE TITLE
Only one tag set per day, despite multiple commits

### ISSUE THREAD

**Issue by tenzap** [validator/validator#2090]
### Summary

There have been multiple commits on 2026-05-02.
I would have expected to see a new tag for each one. Like this 26.5.2.1, 26.5.2.2 ... (according to https://github.com/validator/validator/commit/b51a3e066c8f4bd9b2e63517603117b7487eee8b & https://github.com/validator/validator/pull/2067)
But I only see the tag "26.5.2".

BTW, the [current last CI job](https://github.com/validator/validator/actions/runs/25249365413/job/74038947590) failed on the subsquent step. Would that be the cause for the tag not being set?

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

1.
2.
3.

### Expected behavior

n/a

### Actual behavior

n/a

**Comment by sideshowbarker** [comment]
> BTW, the [current last CI job](https://github.com/validator/validator/actions/runs/25249365413/job/74038947590) failed on the subsquent step. Would that be the cause for the tag not being set?

It think that was a symptom — rather then the direct cause. But anyway I think we have the underlying cause fixed now.

**Comment by tenzap** [comment]
Thanks!
